CHANGELOG.md

```diff
@@ -33,95 +41,55 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Upgrading
 
-#### #205 - Lifecycle callbacks now use `this`
+#### #200 - Attribute lifecycle moved into `properties`.
 
-The following callbacks now use `this` to refer to the element instead of it being passed as the first argument:
-
-- `created`
-- `attached`
-- `detached`
-- `attribute`
-- `events`
-- `template`
-
-For example, this:
-
-```js
-created (element) {
-  element.textContent = 'Hello, World!';
-}
-```
-
-Is now this:
-
-```js
-created () {
-  this.textContent = 'Hello, World!';
-}
-```
-
-####  #208 - `attributes` are no longer granular and is now closer to the spec
-
-- You may now only specify a callback as the `attribute` option.
-- The signature of the callback matches the web component spec.
-  - Use `this` to refer to the element.
-  - Arguments are: `name`, `oldValue` and `newValue` in that order.
-
-Previous:
+The old way of specifying behaviour within properties:
 
 ```js
 attributes: {
-  myAttribute1 (element, changes) {},
+  myAttribute1: function (element, changes) {},
   myAttribute2: {
     value: 'initial value'
   },
   myAttribute3: {
-    value () {
+    value: function () {
       return 'initial value';
     }
   },
   myAttribute4: {
-    created (element, changes) {},
-    updated (element, changes) {},
-    removed (element, changes) {},
+    created: function (element, changes) {},
+    updated: function (element, changes) {},
+    removed: function (element, changes) {},
   }
 }
 ```
 
-Current:
-
-```js
-attribute (name, oldValue, newValue) {
-
-}
-```
-
-To translate the above variations to the new `properties` option, you would do the following:
+In the example above, each attribute would automatically create a property link. All behaviour would be handled by the attribute and there was no disctinction between a property value and an attribute value. This is a little backward, though, since not all API points need, or should, have linkage between properties and attributes. Now, everything gets defined as properties and has an option to be linked to an attribute. To translate the above variations to the new `properties` option, you would do the following:
 
 ```js
 properties: {
   myAttribute1: {
     attr: true,
-    set (newValue, oldValue) {
+    set: function (newValue, oldValue) {
 
     }
   },
   myAttribute2: {
     attr: true,
-    value: 'initial value',
-    set (newValue, oldValue) {
+    init: 'initial value',
```

Comment:
Naming is hard. Happy to discuss alternatives:

- default
- init
- value

Though, `default` is a keyword and it's use is overloaded.